### PR TITLE
Add very basic mutation for updating users

### PR DESCRIPTION
**Pull request by jonallured**
This PR aims to support the work going on over on Forque for editing a user. Prior to this PR there wasn't a way to update a user as an admin but now you can!

```graphql
mutation UpdateUserMutation($input: UpdateUserMutationInput!) {
  updateUser(input: $input) {
    clientMutationId
  }
}

{
  "input": {
    "email":"jon.allured+updated@gmail.com",
    "id":"SHHHH"
  }
}
```

Note that this is wired up to our regular old v1 api endpoint in Gravity for users and so that means it will inherit any access controls already in place there.

### CODE REVIEW

**Review by damassi — APPROVED** [review]
👌

File: src/schema/v2/users/updateUserMutation.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        {} as GravityInput
+      )
+
+    try {
+      return await updateUserLoader?.(args.id, gravityOptions)
+    } catch (err) {
+      if ("body" in (err as any)) {
+        const e = err as GravityError
+        throw new Error(e.body.text ?? e.body.error ?? e.body.message)
+      }
```
Comment (damassi) [inline src/schema/v2/users/updateUserMutation.ts:53]: @jonallured - curious how you landed on this?

Comment (jonallured) [reply src/schema/v2/users/updateUserMutation.ts:53]: LOL I just ripped off @dzucconi on this one:

https://github.com/artsy/metaphysics/blob/main/src/schema/v2/me/updateMyPasswordMutation.ts#L45-L66

Comment (damassi) [reply src/schema/v2/users/updateUserMutation.ts:53]: Ok good. find damon, copy 😄

Comment (jonallured) [reply src/schema/v2/users/updateUserMutation.ts:53]: haha pretty much!

**Pull request merged**